modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -908,6 +895,187 @@ The number of transform processors in a specific state (running, inactive, error
 
 ---
 
+=== Cloud storage metrics
+
+This section provides reference descriptions of the metrics available when Redpanda's xref:manage:tiered-storage.adoc[] is enabled.
+
+include::reference:partial$public_metrics_tip.adoc[]
+
+=== Prerequisites
+
+Cloud storage metrics are only enabled if you have:
+
+- xref:manage:tiered-storage.adoc[] enabled
+- The cluster property xref:reference:properties/cluster-properties.adoc#cloud_storage_enabled[cloud_storage_enabled] is set to `true`
+
+=== redpanda_cloud_storage_cache_space_size_bytes
+
+Sum of size of cached objects
+
+=== redpanda_cloud_storage_housekeeping_drains
+
+Number of times upload housekeeping queue was drained
+
+=== redpanda_cloud_storage_spillover_manifests_materialized_bytes
+
+Bytes of memory used for spilled manifests currently cached in memory
+
+=== redpanda_cloud_storage_cache_op_put
+
+Number of objects written into cache
+
+=== redpanda_cloud_storage_segments
+
+Total number of accounted segments in the cloud for the topic
+
+=== redpanda_cloud_storage_jobs_local_segment_reuploads
+
+Number of segment reuploads from local data directory
+
+=== redpanda_cloud_storage_cache_trim_failed_trims
+
+Number of times could not free the expected amount of space, indicating possible bug or configuration issue
```

Comment:
This seems to be missing a word or subject -- Number of times Tiered Storage..? Redpanda?

Reply:
yes, Redpanda is the subject
```
Number of times Redpanda could not free the expected amount of space, indicating possible bug or configuration issue
```